addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -308,6 +299,22 @@ private String convertToText(JsonElement element) {
         // }
     }
 
+    private String convertToImageUrl(JsonElement element) {
+        String text = element.getAsString();
+        if (!text.isEmpty()) {
+            String encodedURL;
```

Comment:
Now I understand, thanks for clarification. No, there is no reason, it is just the way Eclipse creates try-catch blocks when using quick fix :-)